http-svr/http_svr.ml

```diff
@@ -407,107 +407,120 @@ let request_of_bio_exn_slow ic =
         []
   in
   let headers = read_rest_of_headers 242 in
-  {
-    req with
-    Request.cookie= Http.parse_keyvalpairs !cookie
-  ; content_length=
-      (if !content_length = -1L then None else Some !content_length)
-  ; auth= !auth
-  ; task= !task
-  ; subtask_of= !subtask_of
-  ; content_type= !content_type
-  ; host= !host
-  ; user_agent= !user_agent
-  ; additional_headers= headers
-  ; accept= !accept
-  }
+  let request =
+    {
+      req with
+      Request.cookie= Http.parse_keyvalpairs !cookie
+    ; content_length=
+        (if !content_length = -1L then None else Some !content_length)
+    ; auth= !auth
+    ; task= !task
+    ; subtask_of= !subtask_of
+    ; content_type= !content_type
+    ; host= !host
+    ; user_agent= !user_agent
+    ; additional_headers= headers
+    ; accept= !accept
+    }
+  in
+  (request, None)
```

Comment:
Why None? can't we pass the proxy_seen through the same we do for request_of_bio on the fast path? I think we have the fast path turned off in quite a few places due to requirement to do file descriptor passing on some of xapi's sockets (and wanting to stop reading exactly where headers finish).

Reply:
The slow path doesn't have code at all yet for parsing the PROXY header, so it doesn't have this bug, and therefore it doesn't need this bug fix. Whether we need to add it to that path is a separate question (and patch).